nat-lab/tests/telio.py

```diff
@@ -657,7 +657,7 @@ async def on_stderr(stdout: str) -> None:
                     if self._process.is_executing():
                         await self.stop_device()
                         self._quit = True
-                    self.get_proxy().shutdown()
+                    self.get_proxy().shutdown(self._connection.target_name())
```

Comment:
It would have been enough to pass `self._node.name` but this works too :smile:

Reply:
oh, I missed it :D

Reply:
oh wait, but is'nt `self._node.name` just normally `alpha` or `beta` or similar? In that case, maybe it is a bit better to have specific container name?

Reply:
I think the most important thing is to be able to see if the shutdown is happening multiple times on the same client, which the node name would tell us, but your method would make it easier to perhaps see patterns over time so let's go with it